lib/Common/CommonPal.h

```diff
@@ -274,6 +274,10 @@ inline T InterlockedDecrement(
     return __sync_sub_and_fetch(Addend, T(1));
 }
 
+inline __int64 _abs64(__int64 n)
```

Comment:
We haven't used any STL in chakracore for various reasons. Maybe in future we could reevaluate.